src/platform/haswell/platform.c

```diff
@@ -170,7 +170,8 @@ int platform_init(struct sof *sof)
 
 	/* clear mailbox for early trace and debug */
 	trace_point(TRACE_BOOT_PLATFORM_MBOX);
-	bzero((void *)MAILBOX_BASE, IPC_MAX_MAILBOX_BYTES);
+	trace_point(MAILBOX_BASE);
```

Comment:
Is it a debug call and you forgot to remove it?

Reply:
yep :)